contrib/test_jenkins.sh

```diff
@@ -684,9 +711,7 @@ run_coverity() {
 #
 run_gtest() {
 
-	#load cuda modules if available
-	module_load dev/cuda || true
-	module_load dev/gdrcopy || true
+	try_load_cuda_env
```

Comment:
I didn't unload because we redone the "make" without reconfigure  in run_coverity() after run_gtest(), which causing mismatch conf if I unload in run_gtest.
Anyway, I moved cuda env globally for all the tests/build. 
BTW, I squashed and force pushed the PR.